rag_eval/evals.py

```
@@ -0,0 +1,914 @@
+
+
+# === Evaluation Classes ===
+
+
+class EarthdataEvaluator:
+    """Evaluator for Earthdata RAG system using Ragas metrics and Langfuse experiments"""
+
+    def __init__(
+        self,
+        mcp_server_url: str,
+        llm_model: str = "amazon.nova-pro-v1:0",
```

Ruff (0.15.1)

[warning] Unused method argument: `llm_model`

(ARG002)